fixbackend/app.py

```diff
@@ -69,6 +69,10 @@ async def setup_teardown_application(_: FastAPI) -> AsyncIterator[None]:
         deps.add(SN.graph_db_access, GraphDatabaseAccessManager(cfg, session_maker))
         client = deps.add(SN.inventory_client, InventoryClient(cfg.inventory_url))
         deps.add(SN.inventory, InventoryService(client))
+        deps.add(
+            SN.cloudaccount_publisher,
+            RedisPubSubPublisher(readwrite_redis, "fixbackend::cloudaccount", f"fixbackend-{cfg.instance_id}"),
```

Comment:
You used the PubSub Publisher.
That one uses ephemeral pubsub messaging.
Please use the StreamPublisher